### ISSUE TITLE
TPROC-H schema for Postgres potentially missing crucial indexes

### ISSUE THREAD

**Issue by pstef** [TPC-Council/HammerDB#192]
The test runs much better once I create these indexes:
```
CREATE INDEX linteitem_orderkey_btree ON lineitem (l_orderkey);
CREATE INDEX linteitem_suppkey_btree ON lineitem (l_suppkey);
CREATE INDEX supplier_nationkey_btree ON supplier (s_nationkey);
CREATE INDEX orders_orderstatus_btree ON orders (o_orderstatus);
CREATE INDEX part_brand_btree ON part (p_brand);
CREATE INDEX part_container_btree ON part (p_container);
CREATE INDEX lineitem_partkey_btree ON lineitem (l_partkey);
```
They mostly cover columns on which tables are joined in some queries. From what I understand, Postgres doesn't create indexes for foreign key constraints - unlike MySQL.

**Comment by sm-shaw** [comment]
Yes this is correct and the reason is correct as well, the foreign keys were created as per other databases however PostgreSQL also needs the indexes creating. This was raised in Issue #164 and fixed in PR #170. The indexes will be included from release 4.1.

**Comment by sm-shaw** [comment]
Closed as duplicate of Issue #164, fix already accepted.
